templates/rhba70-kieserver-basic-s2i.yaml

```diff
@@ -0,0 +1,307 @@
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    description: Application template for Red Hat Business Automation Execution Server 7.0 application built using S2I.
+    iconClass: icon-jboss
+    tags: rhba,jboss,xpaas
+    version: 1.4.0
```

Comment:
wouldn't be better start the xpaas with the current version?

Reply:
Gonna keep it at 1.4.0 for now, and update it just before we tag.

Reply:
+1